wafer/talks/models.py

```diff
@@ -169,7 +169,9 @@ class Meta:
     notes = models.TextField(
         _("notes"),
         null=True, blank=True,
-        help_text=_("Any notes for the conference organisers?  "
+        help_text=_("Any notes for the conference, e.g. why you are in a good "
+                    "position to present on this topic, if it is not "
+                    "clear from your bio already. "
```

Comment:
I agree that more explanation is needed, but I'm a bit -1 on this wording. It feels too much like "please justify yourself" to me?

Reply:
How about:

> Any notes for the conference? Such as additional background on the topic or presenters that isn't going to be published publicly, special requirements for the event, or thoughts on scheduling.

Reply:
Sounds good!